src/patterns/scheduled-lambda.ts

```diff
@@ -48,8 +50,11 @@ import type { GuFunctionProps } from "../constructs/lambda";
  *  monitoringConfiguration: { noMonitoring: true }
  * ```
  */
-export interface GuScheduledLambdaProps extends Omit<GuFunctionProps, "rules" | "apis" | "errorPercentageMonitoring"> {
-  schedule: Schedule;
+export interface GuScheduledLambdaProps extends Omit<GuFunctionProps, "apis" | "errorPercentageMonitoring"> {
+  rules: Array<{
+    schedule: Schedule;
+    description: string;
```

Comment:
I would make this description optional. In many cases (e.g. the [example in the tests](https://github.com/guardian/cdk/pull/481/files#diff-8321919ed44468c832ea1017655c47c39ca6fe9943c8a84e56aab6e68e207988R19)), I don't find that providing a description adds any value.

Reply:
Done.